### DISCUSSION TITLE
Lag time between starting nunit and tests actually running

### DISCUSSION THREAD

**Discussion by lpanger** [OpenCover/opencover#1010] [Q&A]
My nunit test suite takes about 2 minutes to run without opencover. When started with opencover, my CPU usage is at 0% for about 5 minutes before the tests start up (I'm assuming this by monitoring the CPU usage). Does anyone know what is happening during this time? Is it possible to speed it up?

**Comment by sawilde** [comment]
Sorry I don't see that myself with my tests - it may be the profiler+runtime taking extra time when but without seeing your specific example I can't say. 

You can try running DebugView at the same time and see if anything useful pops out through that logging.
